docs/reference/public-api.md

```diff
@@ -0,0 +1,70 @@
+---
+id: public-api
+title: Camunda 8 public API
+sidebar_label: Public API
+description: Understand the Camunda 8 public API, its stability guarantees under Semantic Versioning (SemVer), and the policies governing API changes and versioning.
+---
+
+Camunda 8 follows [Semantic Versioning (SemVer)](https://semver.org/) to provide users with a stable and reliable platform. A central requirement of SemVer is a clearly defined public API. This page defines what is included in Camunda 8's public API, outlines our versioning policies, and explains what you can expect when upgrading.
+
+:::note
+The term "public API" in this context refers to the SemVer definition of interfaces with stability guarantees, not to be confused with external APIs that are available to users.
+:::
+
+## What is the public API?
+
+The public API represents the official contract between Camunda and its users for SemVer stability. We guarantee that no breaking changes will be introduced to the public API in minor or patch releases. You can build on these interfaces with confidence, knowing they are stable and backwards-compatible.
+
+Note that this is a subset of all available APIs — many [APIs](/apis-tools/working-with-apis-tools.md) are available for external use but are not included in the formal public API stability commitment.
+
+Only the components explicitly listed on this page (see ["Included in the public API"](#included-in-the-public-api)) are considered part of the public API. Anything not listed is _not_ covered by SemVer stability guarantees.
+
+The public API contract is introduced with the 8.8 release.
+
+### Included in the public API
+
+The following components are officially part of the Camunda 8 public API:
+
+- The [orchestration cluster API](/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md), which provides the primary interface for interacting with the orchestration cluster.
+- [Camunda Process Test](/apis-tools/testing/getting-started.md), which provides testing capabilities for process automation scenarios.
+- [Camunda Java client](/apis-tools/java-client/index.md), which provides Java libraries for interacting with Camunda 8.
+- [Camunda Spring Boot SDK](/apis-tools/spring-zeebe-sdk/getting-started.md), which provides Spring Boot integration for Camunda 8.
```

Comment:
🔧  to align it with the wording for the client, given the sdk is a integration layer for the client into spring
```suggestion
- [Camunda Spring Boot SDK](/apis-tools/spring-zeebe-sdk/getting-started.md), which provides Spring Boot integration for interacting with Camunda 8.
```